<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5">
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <widget class="QLabel" name="image_label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>821</width>
               <height>621</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>随机风格</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>人物风格</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>未来科技</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>艺术风格</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>保留位</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>清除显示</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" colspan="2">
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="contextMenuPolicy">
                   <enum>Qt::NoContextMenu</enum>
                  </property>
                  <property name="text">
                   <string>图片生成</string>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="default">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
